docs/embeddable-uis/element/access-requests.mdx

```diff
@@ -0,0 +1,589 @@
+</CodeDropdown>
+
+### Login Errors
+
+There are a few possible errors you might find yourself come across while working with the embedding of Permit Elements.
+
+| Error                      | Description                                                                                                                                                                                                                                        |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `USER_NOT_FOUND`           | This error can appear if you are trying to log in as a particular user, but that **user does not exist within Permit**. You can also experience this login error when you are trying to login a user into a **tenant that he does not belong to**. |
+| `TENANT_NOT_FOUND`         | This error can appear when you are passing in a tenant, either in the frontend `permit.elements.login` function, or in your backend URL endpoint, where the tenant has not been created in Permit.                                                 |
+| `INVALID_PERMISSION_LEVEL` | This error will emerge when you are trying to access part of the Element which you have not been given access too. This usually means the role that the person obtained has remained in the **Hidden Roles** permission level.                     |
+| `FORBIDDEN_ACCESS`         | This error will emerge when you are trying to login to an Element you have not been given permission to see.                                                                                                                                       |
+
+## API requests
+
+To use the Access Requests API, follow these steps:
+
+First of all we need to [get our `API_SECRET_KEY`](/api/api-with-cli/#get-your-api-key) from the dashboard and [get the current `project_id` and `env_id`](/api/examples/get-project-and-env)<br/>
+Replace `API_SECRET_KEY` with the key from the Permit dashboard along with the `project_id` and `env_id` you got from the API in the following commands.
+
+### Creating an Access Request
+
+To create a new access request, make a POST request to the following with the required data.
+
+```bash
+curl 'https://api.permit.io/v2/facts/{project_id}/{env_id}/access_requests' \
+  -H 'element_id: ELEMENTS_CONFIG_ID'
+  -data-raw
+ {
+    "access_request_details": {
+        "tenant": "34f5c98e-f430-457b-a812-92637d0c6fd0",
+        "resource": "4d5215ed-38bb-48ed-879a-fdb9ca58522f",
+        "resource_instance": "2d98d9f8-e1b7-4f1d-baad-2edbf6fa6c66",
+        "role": "ac4e70c8-d5be-48af-93eb-760f58fc91a9",
+    },
+    "time_limit": {
+        "start_time": "2019-08-24T14:15:22Z",
+        "end_time": "2019-09-24T14:15:22Z",
+    },
+    "reason": "done onboarding",
+ } \
+```
```

Comment:
Pretty sue that won't work, you don't provide the cookie from before - when you login using `permit.elements.login` your users get a cookie - use this cookie to call the access requests API.

Reply:
fixed